lib/trino-filesystem-alluxio/src/test/java/io/trino/filesystem/alluxio/TestAlluxioFileSystem.java

```diff
@@ -85,11 +89,4 @@ void setup()
         // https://github.com/trinodb/trino/discussions/21969
         System.setProperty("ReportLeakedContainers.disabled", "true");
     }
-
-    @Test
-    void testContainer()
-    {
-        assertThat(ALLUXIO_MASTER_CONTAINER.isRunning()).isTrue();
-        assertThat(ALLUXIO_WORKER_CONTAINER.isRunning()).isTrue();
```

Comment:
`ALLUXIO_WORKER_CONTAINER` is unused now. Please consider refactoring the test in follow-up.

Reply:
The worker is need. The master is for metadata & worker for data.